pombola/kenya/views.py

```diff
@@ -45,6 +50,19 @@
             'g': ('m', 'f'),
             'agroup': ('under', 'over'),
         },
+    },
+    'youth-employment-bill': {
+        'session_key_prefix': 'MIT3',
+        'base_view_name': 'youth-employment',
+        'pageview_label': 'youth-employment',
+        'template_prefix': 'youth',
+        'experiment_key': settings.YOUTH_EMPLOYMENT_BILL_EXPERIMENT_KEY,
+        'qualtrics_sid': 'SV_5hhE4mOfYG1eaOh',
```

Comment:
Oops, lazy copy and paste. Will change it to None to avoid accidentally serving the wrong survey